### PR TITLE
Update 5-creating.md

### PR DESCRIPTION
**Pull request by GreatGhostsss**
Light editing—was pretty clean.

### CODE REVIEW

File: tf-101/5-creating.md [hunk 1]
```diff
@@ -90,47 +89,47 @@ resource "oci_core_vcn" "tf_101" {
...
 
-As mentioned previously, make sure that you specify the correct region you wish to deploy resources into.  This example is using the `us-phoenix-1` region, however you might need to modify this for your needs.  See the [OCI regions documentation](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm) for more information.
+As mentioned previously, ensure that you specify the correct region you wish to deploy resources into. This example is using the `us-phoenix-1` region, however you might need to modify this for your needs. See the [OCI regions documentation](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm) for more information.
 
 ### Resource definition
 
-A Virtual Cloud Network (VCN) is an OCI resource that is required by many other OCI services.  It’s a core, foundational resource in many OCI cloud environments, providing a logical network definition.  Because it’s needed before you can deploy Subnets, Compute Instances, Load Balancers, etc. you'll start with creating a VCN.  As a bonus, VCNs don’t cost a penny in OCI, so this minimizes your chances of incurring charges as you experiment with Terraform on OCI.
```
Comment (ttscoff) [inline tf-101/5-creating.md:117]: I think these actually should be capitalized, as they're OCI services.

File: tf-101/5-creating.md [hunk 2]
```diff
@@ -90,47 +89,47 @@ resource "oci_core_vcn" "tf_101" {
...
 
 Make sure to set the the `compartment_id` value to the OCID of the compartment you wish to use!
 
 ## Setting up OCI Authentication
 
-If we were using Terraform outside of OCI Cloud Shell, we'd need to worry about how Terraform will authenticate with the OCI API.  By running it inside of Cloud Shell, we don't need to worry about these details - it'll be authenticated automatically for us!
+If we were using Terraform outside of OCI Cloud Shell, we'd need to worry about how Terraform will authenticate with the OCI API. By running it inside of Cloud Shell, we don't need to worry about these details—it'll be authenticated automatically for us!
```
Comment (ttscoff) [inline tf-101/5-creating.md:124]: Please change this to `details -- it'll`

File: tf-101/5-creating.md [hunk 3]
```diff
@@ -90,47 +89,47 @@ resource "oci_core_vcn" "tf_101" {
...
+Terraform needs different files to be able to be able to properly function reliably. This is largely hidden out of view so that you don’t need to be mired down in the nitty-gritty details. It’s still a good idea to have a high-level idea of what’s going on, so let's touch on a few things that Terraform needs.
 
-Terraform doesn’t ship with any providers, so one of the first things that Terraform does is to examine any referenced providers and then download them as needed.  In this case, we've told Terraform to use the OCI Terraform provider, but it doesn’t have this provider – yet.  This is taken care of during the Terraform initialization process.
+Terraform doesn’t ship with any providers, so one of the first things that Terraform does is to examine any referenced providers and then download them as needed. In this case, we've told Terraform to use the OCI Terraform provider, but it doesn’t have this provider—yet. This is taken care of during the Terraform initialization process.
```
Comment (ttscoff) [inline tf-101/5-creating.md:130]: Same, double dash surrounded by spaces.

File: tf-101/5-creating.md [hunk 4]
```diff
@@ -25,48 +25,47 @@ redirect: https://developer.oracle.com/tutorials/tf-101/5-creating/
...
 {% imgx aligncenter assets/terraform-101.png 400 400 "Terraform 101" "Terraform 101 Tutorial Series" %}
 
-The next few lessons in this tutorial will take you through an adventure where resources are created, modified, and then finally destroyed using Terraform.  While you got a little taste for this in the [Experiencing Terraform lesson](2-experiencing-terraform.md), we'll cover the topics in greater detail here, pointing out more details along the way.
+The next few lessons in this tutorial will guide you through an adventure where resources are created, modified, and then finally destroyed using Terraform. While you got a little taste for this in the [Experiencing Terraform lesson](2-experiencing-terraform.md), we'll cover the topics in greater detail here, pointing out more details along the way.
 
-Let's start using Terraform on Oracle Cloud Infrastructure (OCI) by deploying a Virtual Cloud Network (VCN), a foundational OCI resource in which you can deploy other OCI resources to.
+Let's start using Terraform on Oracle Cloud Infrastructure (OCI) by deploying a virtual cloud network (VCN), a foundational OCI resource in which you can deploy other OCI resources to.
```
Comment (ttscoff) [inline tf-101/5-creating.md:30]: Virtual Cloud Network is a service, right? As such, should it be capitalized?

File: tf-101/5-creating.md [hunk 5]
```diff
@@ -25,48 +25,47 @@ redirect: https://developer.oracle.com/tutorials/tf-101/5-creating/
...
 
 ## Prerequisites
 
-You should have an Oracle Cloud Infrastructure (OCI) account setup.  [Click here]({{ site.urls.alwaysfree }}) to create a new cloud account.
+You should have an OCI account setup. [Click here]({{ site.urls.alwaysfree }}) to create a new cloud account.
```
Comment (ttscoff) [inline tf-101/5-creating.md:34]: Only looking at a diff here not the entire file, but if OCI isn't already expanded on first use it should be. But that might have already happened, in which case ignore this comment.

**Review by ttscoff — CHANGES_REQUESTED** [review]

File: tf-101/5-creating.md (hunk 5 above)
Comment (GreatGhostsss) [reply tf-101/5-creating.md:34]: Yup, it was on line 30

**Review by ttscoff — APPROVED** [review]

**Pull request merged**